blockbot/blockbot.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Name: blockbot
+ * Description: Blocking bots based on detecting bots/crawlers/spiders via the user agent and http_from header.
+ * Version: 0.1
+ * Author: Philipp Holzer <admin@philipp.info>
+ *
+ */
+
+use Friendica\App;
+use Friendica\Core\Hook;
+use Friendica\Core\System;
+use Jaybizzle\CrawlerDetect\CrawlerDetect;
+
+require_once __DIR__ . DIRECTORY_SEPARATOR . 'vendor' . DIRECTORY_SEPARATOR . 'autoload.php';
+
+function blockbot_install() {
+	Hook::register('init_1', 'addon/blockbot/blockbot.php', 'blockbot_init_1');
```

Comment:
I suggest using `__FILE__` instead of `'addon/blockbot/blockbot.php'`.

Reply:
But `__FILE__` creates a absolute path, would this be a problem? Because currently we're just saving relative paths

Reply:
It used to be, but we now strip Friendica basedir from the file path if it's absolute.